compiler/cpp/src/generate/t_java_generator.cc

```diff
@@ -323,6 +326,7 @@ class t_java_generator : public t_oop_generator {
 
   // Generate FutureIface (based on CompletableFuture)
   bool future_iface_;
+  bool mixed_futures_;
```

Comment:
Can you please add a comment about what the parameter does?
Also please add it to the command line help, below in this file.